README.rst

```diff
@@ -0,0 +1,36 @@
+Nanoshield_Thermocouple
+=======================
+
+This is the Arduino software library to access the Thermocouple Nanoshield (MAX31855 IC) from Maxim Integrated.
+
+* Source code: https://github.com/circuitar/Nanoshield_Thermocouple
+* Documentation: http://nanoshield-thermocouple.readthedocs.org/
+* Reference board: 'Termopar Nanoshield'_ from Circuitar_.
+
+Library features include:
+
+* Get internal temperature.
+* Get external temperature.
+* Check errors (shorted to vcc, shorted to ground and open circuit).
+
+To install, just click **Download ZIP** and install it using **Sketch > Include Library... > Add .ZIP Library** in the Arduino IDE.
+
+----
+
+The following examples_ are provided:
+
+- MultiThermometer_ read temperature from multiple Thermocouple Nanoshields.
+- SimpleThermometer_ simple serial port thermometer application.
+
+.. _'Termopar Nanoshield': https://www.circuitar.com.br/nanoshields/modulos/termopar/
```

Comment:
On Github, let's use the link to the board page in circuitar.com instead of circuitar.com.br.